gui/default/assets/css/overrides.css

```diff
@@ -569,6 +569,14 @@ html[lang|="ko"] i {
     user-select: all;
 }
 
+#login-options .panel-body {
+  padding: 1em;
+}
+
+#login-options .stay-logged-in {
+  margin-top: 0.25em;
+}
```

Comment:
Ok, but I don't know what would be useful to put in such a comment. It just looks better to me this way? (and I didn't find any suitable pre-baked spacing classes in [Bootstrap CSS](https://getbootstrap.com/docs/3.3/css/))

Reply:
The comment should state where this override applies.  I have no idea what it does, as the CSS classifiers don't speak for themselves and it would be better not having to search the HTML code to see where this applies.

Reply:
Ok. This made me look more into why the padding style is needed, and I discovered [this global padding decrease class](https://github.com/emlun/syncthing/blob/13d545c9f2e54ac32e35047a475a3f4bbd616e70/gui/default/assets/css/overrides.css#L264-L266). Since that style is no longer appropriate for all collapsible panels, I renamed it to `.panel-body.less-padding` and added the `less-padding` class everywhere it currently applies. Hopefully that name should be self-explanatory enough that its comment is no longer needed.

Also added a comment to the remaining `#login-options .stay-logged-in` style in 709cbc121bae3c0015b134e081d01d2e1291e516.